### DISCUSSION TITLE
Can someone implement GAN in TorchSharp

### DISCUSSION THREAD

**Discussion by ad48hp** [xamarin/TorchSharp#1283] [Ideas]
Hello,
can someone implement a basic GAN (2014) or its followers such as DCGAN in TorchSharp ?
I tried to do so, but it's quite hard to get it to work.

**Comment by GeorgeS2019** [comment]
### GAN
https://github.com/Yangyangii/GAN-Tutorial

### DCGAN
https://pytorch.org/tutorials/beginner/dcgan_faces_tutorial.html
